src/btree/bt_split.c

```diff
@@ -12,10 +12,8 @@
  * Tuning; global variables to allow the binary to be patched, we don't yet have
  * any real understanding of what might be useful to surface to applications.
  */
-static u_int __split_deepen_max_internal_image = 100;
-static u_int __split_deepen_min_child = 10;
-static u_int __split_deepen_per_child = 100;
-static u_int __split_deepen_split_child = 100;
+static u_int __split_deepen_min_child = 10000;
+static u_int __split_deepen_per_child = 1000;
```

Comment:
It seems pointless to split a page (even a big page), if there's nothing on it to split.